test/helper.rb

```diff
@@ -1,28 +1,13 @@
-require 'rubygems'
-require 'bundler'
+$LOAD_PATH.unshift(File.expand_path("../../", __FILE__))
+require "test-unit"
+require "fluent/test"
+require "fluent/test/helpers"
 
-require 'test/unit'
-require 'test/unit/rr'
+Test::Unit::TestCase.include(Fluent::Test::Helpers)
+Test::Unit::TestCase.extend(Fluent::Test::Helpers)
 
-$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
-$LOAD_PATH.unshift(File.dirname(__FILE__))
-
-require 'fluent/test'
-
-class Test::Unit::TestCase
-  def capture_log(log)
-    if defined?(Fluent::Test::TestLogger) and log.is_a?(Fluent::Test::TestLogger) # v0.14
-      yield
-      log.out.logs.join("\n")
-    else
-      begin
-        tmp = log.out
-        log.out = StringIO.new
-        yield
-        return log.out.string
-      ensure
-        log.out = tmp
-      end
-    end
+def normalize_logs(logs)
+  logs.collect do |log|
+    log.gsub(/\A\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [-+]\d{4} \[info\]: /, "") 
   end
 end
```

Comment:
We should not define a method in top level.

We should define a helper module, define helper methods in it and include the helper module in each our `*Test`.

Reply:
Oh, thanks for the advice.
I will fix it.

Reply:
Fixed.

https://github.com/fluent-plugins-nursery/fluent-plugin-flowcounter-simple/compare/e5dc5e920f7d7862be9574fba60d280726fed513..55b4cf4093e976a26648c869f4060012e6f9a210